src/index.ts

```diff
@@ -93,7 +101,26 @@ export interface LintOptions {
    * Warnings to ignore. After the rules are run, any warning that matches
    * one of these codes is ignored, project-wide.
    */
+  warningsToIgnore?: string[];
+
+  /**
+   * Awkward way of spelling the `warningsToIgnore` lint option.
+   *
+   * Used only if `warningsToIgnore` is not specified.
```

Comment:
Safe to add a "deprecated in favor of..." mention in the property description, even if an actual warning message is coming in a later version.